app/build.gradle

```diff
@@ -1,11 +1,10 @@
 plugins {
```

Comment:
Note: In this file I've only added one dependency for Kotlin coroutines and changed all single- to double quotes